app/lib/ui/metapolator/parameter-panel/selection/selection-parameter/selection-parameter-controller.js

```diff
@@ -82,34 +106,49 @@ define([
             return newValue;
         }
 
-        function manageInputValue(value, parameter, operator, event) {
-            // Adjust the value based increment `step` and `event` key
-            var currentValue = value.current
-              , step = parameter.base.step;
-            if (typeof (currentValue) === "string") {
-                // Try to make a number out of the string.
-                currentValue = parseFloat(currentValue.replace(',', '.'));
-            }
-            if (!isNumeric(currentValue)) {
-                return value.fallback;
+        var CPSError = errors.CPS;
```

Comment:
move this line way up in the file, just before the line 

```
function SelectionParameterController($scope, metapolatorModel, project) {
```

It's like a specification of the module that was loaded and it's interesting to see this very early.

Alternatively, we should (maybe much rather) export the `evaluateMathExpression` function from the simpleMathEngine module. It's kind of too specialized to be defined here but a good fit for there.